src/components/form/HdTextarea.vue

```diff
@@ -171,8 +185,25 @@ export default {
 </script>
 
 <style scoped lang="scss">
+@import 'homeday-blocks/src/styles/mixins.scss';
+
 .textarea {
   display: block;
   resize: vertical;
 }
+.maxlength-indicator {
+  position: absolute;
+  right: 0;
+  bottom: -#{($sp-s * 2) + ($sp-xs / 2)};
+  @include font("DS-60");
+  padding: 0px $sp-s;
+  color: getShade($neutral-gray, 70);
```

Comment:
```suggestion
  color: getShade($quaternary-color, 70);
```

Reply:
oh, i forgot about that! :d thanks!

Reply:
https://github.com/homeday-de/homeday-blocks/pull/815/commits/504d02ceac39fc5ee2fdceb9ef2a76471a98a41c